postgres-appliance/Dockerfile

```diff
@@ -406,7 +410,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 \
         && find /usr/share/python-babel-localedata/locale-data -type f ! -name 'en_US*.dat' -delete \
 \
-        && pip3 install filechunkio wal-e[aws,google,swift]==$WALE_VERSION kubernetes==3.0.0 \
+        && pip3 install filechunkio wal-e[azure,aws,google,swift]==$WALE_VERSION kubernetes==3.0.0 \
```

Comment:
I wouldn't put "azure" into extras, it installs 247M of useless crap...
It should be enough to explicitly install "azure-storage".

Reply:
I tried but I got this message:
`wal-e 1.1.0 does not provide the extra 'azure-storage'`